default/python/universe_generation/teams.py

```diff
@@ -0,0 +1,114 @@
+from logging import warning, debug
+
+from common.configure_logging import redirect_logging_to_freeorion_logger
+
+# Logging is redirected before other imports so that import errors appear in log files.
+redirect_logging_to_freeorion_logger()
+
+import freeorion as fo
+
+from typing import Dict, Tuple, List
+
+from operator import itemgetter
+from empires import home_system_layout
+from util import iterate_pairs
+
+
+def home_system_team_core(home_systems: List[int], teams: List[Tuple[int, int]]) -> Dict[int, int]:
+    """
+    Choose core for teams.
+    Returns map from team to core home system.
```

Comment:
You can drop a couple of words about teams here.  Is it (teams, empire_id) or (empire_id, teams)?

It is not oblivious that the team means. 

My expectation was that each game has teams: number of teams == number of the empiers (everyone against everyone), minimum number of teams is 2.   

But looks like team is additional grouping.  We can have 3 empires (1,2,3) and one team ([1]).

Reply:
Team is a two or more empires shares same non-negative `starting_team`.

Reply:
Added description what data is in `teams` parameter.